src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -363,13 +364,60 @@ public void run(final RunNotifier notifier) {
             statement.evaluate();
         } catch (AssumptionViolatedException e) {
             testNotifier.addFailedAssumption(e);
-        } catch (StoppedByUserException e) {
```

Comment:
Now `StoppedByUserException` is handled in the `processExecutionFailures()` method, despite single or multiple exceptions came from evaluating class block statement.